templates/necto/project_templates/3_library_sdk/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.19)
 
+include(mikroeUtils)
```

Comment:
Move it above `set(PROJECT_TYPE "${PROJECT_TYPE_VALUE}" CACHE STRING "" FORCE)`

Simply to be consistent with other template placing.